---
layout: page
title: CBB752 Spring 2017
tagline: Final Project
---

Project Title
------------------


Table of Contents
-----------------------




**Contributors**
 -Writing: Yuyang Liu  

 -Coding: Zhaolong Yu

 -Pipeline: Megan Brady


### Introduction:
functional implications of SNP  
This project aims at predicting the deleteriousness of SNVs in Carl’s genome. Single nucleotide variants in the coding region of the DNA sequence are likely to affect the structure of the protein product and lead to observable impacts on physiological functioning and disease propensity. Coding region mutations could be classified as silent, missense, nonsense or frameshift mutations based upon the changes in codons and open reading frame. However, deleteriousness could vary significantly among mutations of the same category, and an applicable method to assess deleteriousness must rely upon a more comprehensive prioritization scheme. The standard for determining mutation deleteriousness could either be based upon biochemical or evolutionary information. Bioinformatics tools such as VIPUR incorporates both structural and evolutionary information as the training dataset of the program (Baugh et al., 2016). This strategy allows better analysis of rare proteins with seldom any homology while avoid the oversimplified assumptions of a purely structural approach. However, due to the limited scope for this project, we’ve chosen a first-principle approach that doesn’t involve extensive alignment with homologues. 




### Writing:








### Coding:


#### Documentation:


#### Results:







### Pipeline:


#### Documentation:


#### Results:









#### Conclusions:








#### References:

 References can be included here or at the end of each relevant section.
 
 
